docs/samples/Microsoft.ML.Samples/Dynamic/FeatureSelectionTransform.cs

```diff
@@ -48,14 +48,14 @@ public static void FeatureSelectionTransform()
             // In this example we define a CountFeatureSelectingEstimator, that selects slots in a feature vector that have more non-default 
             // values than the specified count. This transformation can be used to remove slots with too many missing values.
             var countSelectEst = ml.Transforms.FeatureSelection.SelectFeaturesBasedOnCount(
-                inputColumn: "Features", outputColumn: "FeaturesCountSelect", count: 695);
+                outputColumnName: "FeaturesCountSelect", sourceColumnName: "Features", count: 695);
```

Comment:
Were we preferring the name "input" compared to "source" for the column names? Or is this exceptional for some reason?

Reply:
this is sourceColumnName in the 6th iteration. 

---
In reply to: [251532233](https://github.com/dotnet/machinelearning/pull/2239#discussion_r251532233) [](ancestors = 251532233)

Reply:
> i think we agreed to sourceColumnName..

Are you sure @sfilipi ? Perhaps I am misinterpreting your summary of #2064, but it seems fairly clear to me. is this different for some other reasons?

> the name of the column resulting from the transformation will be "outputColumnName" and the name of the column/columns to apply the transformation too would be "inputColumnName".

Reply:
> HA! this is how deep you had planted the "source" name in my mind!

😄 Now that I know I have this ability, I solemnly swear to use my special powers only for good.

(My own good, that is.)